docs/ibis-for-dplyr-users.ipynb

```diff
@@ -0,0 +1,774 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "starwars"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Inspecting the dataset with `head()`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Just like in R, you can use `head()` to inspect the beginning of a dataset. You can also specify the number of rows you want to get back by using the parameter `n` (default `n = 5`).\n",
+    "\n",
+    "In R:\n",
+    "\n",
+    "```r\n",
+    "head(starwars) # or starwars |> head()\n",
+    "```\n",
+    "\n",
+    "With Ibis:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "starwars.head(6)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There is no `tail()` in Ibis because most databases do not support this operation.\n",
+    "\n",
+    "Another method you can use to limit the number of rows returned by a query is `limit()` which also takes the `n` parameter."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "starwars.limit(3)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Filtering rows with filter()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Ibis, like dplyr, has the `filter` method to select rows based on conditions.\n",
+    "\n",
+    "With dplyr:\n",
+    "\n",
+    "```r\n",
+    "starwars |>\n",
+    "  filter(skin_color == \"light\")\n",
+    "```\n",
+    "\n",
+    "In Ibis:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "starwars.filter(_.skin_color == \"light\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In dplyr, you can specify multiple conditions separated with `,` that are then combined with the `&` operator:\n",
+    "\n",
+    "```r\n",
+    "starwars |>\n",
+    "  filter(skin_color == \"light\", eye_color == \"brown\")\n",
+    "```\n",
+    "\n",
+    "In Ibis, you can do the same by putting multiple conditions in a list:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "starwars.filter([_.skin_color == \"light\", _.eye_color == \"brown\"])"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In previous code, we used the `_` helper we imported earlier. The `_` is shorthand for the table we referenced at the beginning of the expression (in this case, `starwars`). We could have also written the more verbose form,\n",
```

Comment:
I don't think this is strictly correct. I think it is more accurate to say:
> The `_` is shorthand for the table returned by the previous step in the chained sequence of operations.

Reply:
Good catch, thanks!